src/R/Editor/Application.Test/Fixtures/EditorHostMethodFixture.cs

```diff
@@ -2,50 +2,47 @@
 // Licensed under the MIT License. See LICENSE in the project root for license information.
 
 using System.Threading.Tasks;
-using Microsoft.Common.Core.Services;
 using Microsoft.Common.Core.Shell;
-using Microsoft.Common.Core.Test.Fakes.Shell;
 using Microsoft.Languages.Editor.Application.Core;
 using Microsoft.R.Host.Client;
 using Microsoft.R.Host.Client.Test.Script;
 using Microsoft.R.Support.Help;
-using Microsoft.UnitTests.Core.Mef;
+using Microsoft.UnitTests.Core.XUnit;
 using Microsoft.UnitTests.Core.XUnit.MethodFixtures;
 using static Microsoft.UnitTests.Core.Threading.UIThreadTools;
 
 namespace Microsoft.R.Editor.Application.Test {
+    [AssemblyFixture]
```

Comment:
`EditorHostMethodFixture` isn't an assembly-level fixture - it is method-level fixture, this attribute should be removed